snooty/util.py

```diff
@@ -219,3 +219,15 @@ def option_flag(argument: Optional[str]) -> bool:
         raise ValueError('no argument is allowed; "%s" supplied' % argument)
     else:
         return True
+
+
+def split_domain(name: str) -> Tuple[str, str]:
+    """
+    Split a fully-qualified reStructuredText directive or role name into its (domain, name) pair.
+    For example, "mongodb:ref" becomes ("mongodb", "ref"), while simply "ref" becomes ("", "ref").
+    """
```

Comment:
[nit] Docstring formatting doesn't appear to be consistent across this PR!

Reply:
The great thing about code formatters is that you just Don't Worry about stuff like this.

In this case, black puts """ onto its own line for multiline docstrings.

Reply:
Did some more reading into docstring formatting conventions, and updated this PR appropriately!